src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/CMakeLists.txt

```diff
@@ -8,30 +8,18 @@ add_library (aduc::${target_name} ALIAS ${target_name})
 
 find_package (Parson REQUIRED)
 
-target_sources (${target_name} PRIVATE
-    src/adps_gen2.c
-    src/adps2_mqtt_client_module.c)
+target_sources (${target_name} PRIVATE src/adps_gen2.c src/adps2_mqtt_client_module.c)
 
 target_include_directories (
     ${target_name}
     PUBLIC inc
-    PRIVATE ${PROJECT_SOURCE_DIR}/inc
-            ../inc
-            ../../inc
-            ${ADU_AGENT_SDK_INCLUDES}
-            )
+    PRIVATE ${PROJECT_SOURCE_DIR}/inc ../inc ../../inc ${ADU_AGENT_SDK_INCLUDES})
```

Comment:
Why the need to dip into .. and ../.. ?
Should not need to add line 16 if target_link_libraries targets properly setup in this file and in the targets linking to